MAUI/Popup/modal-window.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Modal Window in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Modal Window support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Modal Window in MAUI Popup (SfPopup)
+
+You can use popup as modal window by using the built-in close icon and the [SfPopup.StaysOpen](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_StaysOpen) property prevents interaction with your application until you close the window.

```

Comment:
You can use the popup as a modal window using the built-in close icon and the [SfPopup.StaysOpen](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Popup.SfPopup.html#Syncfusion_Maui_Popup_SfPopup_StaysOpen) property prevents interaction with your application until you close the window.